### ISSUE TITLE
Thanks for your work.I have a question. Since the prompt in the model's input already describes the content represented by the vectors, why is it necessary to align the vectors with the LLM during the pre-training process? Are the vectors used to help the model further understand the driving scenario based on the prompt? Are the labels in the pre-training process the prompts generated by lanGen? What is the purpose of the 100k question-answer pairs in the pre-training process?

### ISSUE THREAD

**Issue by lilpeng** [wayveai/Driving-with-LLMs#24]
Could you explain what the model's input and the corresponding labels are during the pre-training stage to help me better understand this process?

**Comment by MLgeek7325** [comment]
您好，您的邮件我已收到！

**Comment by melights** [comment]
Thanks for the question. The short answer is that we were trying to proof that we can align the the vectors with LLMs hence we can use the similar method to align the image with LLMs as described in the Lingo work: https://arxiv.org/abs/2312.14115

**Comment by otepencelik** [comment]
Could you please provide a long answer as well? It is not clear to me how the LanGen is employed during pre-training. Are the LanGen outputs used as pseudo ground truth during pre-training and the loss function is applied between those and LLM outputs? Is the LanGen also employed during the fine-tuning stage to create the scene description and include it in the input prompt? @melights
